res/controllers/Hercules DJ Console Mk1.hid.xml

```diff
@@ -7,6 +7,23 @@
         <forums>http://www.mixxx.org/forums/viewtopic.php?f=7&amp;t=4081</forums>
         <manual>hercules_dj_console_mk1</manual>
     </info>
+    <settings>
+        <group label="Backwards Compatibility">
+            <row orientation="vertical">
+                <option
+                    variable="pitchBendAsJogWheel"
```

Comment:
please also document this setting (as well as the new behavior in 2.5.1) in the manual.

Reply:
Currently there is no manual page for this controller. I may or may not be able to find the time to make one since that's a much bigger change, but I'd like to do a few patches first since I only have this controller for ~12 more hours while I do a repair on it.